app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -50,17 +49,66 @@ def show
         end
 
         def update
+          return if wait_for_result?
+
           result = handle_stored_result(
             user: document_capture_session.user,
             store_in_session: false,
           )
+          # TODO: new analytics event?
+          analytics.idv_doc_auth_document_capture_submitted(
+            **result.to_h.merge(analytics_arguments),
+          )
 
           if result.success?
             redirect_to idv_hybrid_mobile_capture_complete_url
           else
             redirect_to idv_hybrid_mobile_socure_document_capture_url
           end
         end
+
+        private
+
+        def wait_for_result?
```

Comment:
This is identical to the method in the standard flow controller, but I don't know that the time out will redirect to the same page or not. If, once the dust settles on this, they are still identical then we can refactor into a concern.